docs/product/issues/issue-details/index.mdx

```diff
@@ -61,16 +61,10 @@ To view other events, you can use the dropdown to skip to the latest or oldest e
 
 ![Trace Navigator](./img/trace-navigator.png)
 
-The trace navigator (which is displayed below the date) is an abbreviated view of the related [trace](/product/sentry-basics/tracing/distributed-tracing) for the current transaction. It displays six types of nodes (if applicable), each representing different groups of the event's trace and visualized as a dot in the timeline:
+The trace navigator (which is displayed below the date) is an abbreviated view of the related [trace](/product/sentry-basics/tracing/distributed-tracing) for the current transaction.
 
-- **Root**: Starting transaction
-- **Ancestor**: Events that are descendants of the root, and are directly related to the current event
-- **Parent**: Event that directly started the current event
-- **This Event**: Node representing the current event being viewed
-- **Children**: Events that the current event directly started
-- **Descendants**: Future generations of events created by children of this event
+The Trace Navigator displays the various events that happeneded during the associated trace. When you hover any of the dots on the trace navigator the developer will see what issue that event belongs to and clicking on it will navigatate to that specific event.
```

Comment:
maybe we should revert back to the copy seen here https://github.com/getsentry/sentry-docs/pull/10042/files#diff-051c4d59160f658294ad60b91e188f36142e24b8f9b1dc77436b547cb47a67a2L49-L55